node_exporter.go

```diff
@@ -110,6 +111,21 @@ func loadCollectors(list string) (map[string]collector.Collector, error) {
 		}
 		collectors[name] = c
 	}
+	for name, enabled := range collectorsEnabled {
```

Comment:
We discussed a different behavior in the issue. See https://github.com/prometheus/node_exporter/issues/216#issuecomment-268357097 and my follow up comment.
If that is something you want to work on, go for it. Otherwise I'd do the change.